operations/helm/charts/alloy/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ This document contains a historical list of changes between releases. Only
 changes that impact end-user behavior are listed; changes to documentation or
 internal API changes are not present.
 
+0.2.0 (2024-04-??)
+------------------
+
+- Add prometheus-operator-crds as a dependency
+
```

Comment:
Hey @itspooya thanks a lot for the PR! I think this makes sense and I'll leave the fial word to @\rfratto but adding two small comments

We're actually not releasing a new Helm version for _every_ change but prefer to batch them and use our tooling to release on demand. 

```suggestion
Unreleased
----------

- Add prometheus-operator-crds as a dependency

```